watertap/data/techno_economic/default_case_study.yaml

```diff
@@ -0,0 +1,89 @@
+base_currency: MUSD_2018
+base_period: year
+defined_flows:
+  electricity:
+    value: 0.0595
+    units: USD_2019/kWh
+  alum:
+    value: 0.69
+    units: USD_2020/kg
+    purity: 0.17
+  ammonia:
+    value: 0.79
+    units: USD_2020/kg
+    purity: 1
+  anti-scalant:
+    # Hydrazine
+    value: 3.49
+    units: USD_2020/kg
+    purity: 0.85
+  caustic_soda:
+    value: 0.11
+    units: USD_2020/kg
+    purity: 1
+  #coagulant:
+  #  value: 0.69
+  #  units: USD_2020/kg
+  #  purity: 0.17
+  ferric_chloride:
+    value: 0.88
+    units: USD_2020/kg
+    purity: 1
+  hydrochloric_acid:
+    value: 0.17
+    units: USD_2020/kg
+    purity: 1
+  lime:
+    value: 0.15
+    units: USD_2020/kg
+    purity: 1
+  sodium_bisulfite:
+    value: 0.36
+    units: USD_2020/kg
+    purity: 1
+  sulfuric_acid:
+    value: 0.12
+    units: USD_2020/kg
+    purity: 1
+global_parameters:
+  plant_lifetime:
+    value: 30
+    units: year
+  utilization_factor:
+    value: 1
+    units: dimensionless
+  land_cost_percent_FCI:
+    value: 0.07
```

Comment:
```suggestion
    value: 0.0015
```

Reply:
I see where you get these values- as the initialized values for the Var() in WT3... but they are fixed to something somewhere else. I know it is confusing so apologies